numba/cuda/tests/cudapy/test_constmem.py

```diff
@@ -186,9 +186,15 @@ def test_const_record_optimization(self):
                 self.assertIn('ld.const.v2.f64', jcuconst.ptx,
                               'load record fields as vector of 2x f64')
         elif nvvm70_runtime:
-            # Load of the x and y fields fused into a single instruction
-            self.assertIn('ld.const.v2.u64', jcuconst.ptx,
-                          'load record fields as vector of 2x u64')
+            if windows:
+                # Two ld.const.u32 as above, but using a bit-field insert to
+                # combine them
+                self.assertIn('ld.const.u32', jcuconst.ptx,
+                              'load record fields as u32')
+            else:
+                # Load of the x and y fields fused into a single instruction
+                self.assertIn('ld.const.v2.u64', jcuconst.ptx,
+                              'load record fields as vector of 2x u64')
```

Comment:
Are the `old_runtime` and `nvvm70_runtime` branches now effectively the same?

Reply:
Looks like it - the generated code is different between the old and nvvm70 runtimes, but the check we use for both is the same (because we don't then look for the shift/or vs bfi instructions). I could merge these two branches?

Reply:
Whoops, they're not quite the same - on non-Windows, `ld.const.v2.f64` is used for the old runtime, whereas `ld.const.v2.u64` is used with the NVVM70 one (`f64` vs `u64`).

Reply:
ah, thanks, missed the f64 vs u64

Reply:
It's easy to miss! I only saw it when I collapsed the two branches and saw it start to fail :-)